configure.ac

```diff
@@ -957,8 +946,30 @@ AM_CONDITIONAL([ENABLE_PLUGIN_PERF], [test "${enable_plugin_perf}" = "yes"])
 # -----------------------------------------------------------------------------
 # ebpf.plugin
 
+PKG_CHECK_MODULES(
+    [LIBELF],
+    [libelf],
+    [have_libelf=yes],
+    [have_libelf=no]
+)
+OPTIONAL_BPF_CFLAGS="${LIBELF_CFLAGS}"
+OPTIONAL_BPF_LIBS="${LIBELF_LIBS}"
+
+AC_CHECK_TYPE(
+    [struct bpf_prog_info],
+    [have_bpf=yes],
+    [have_bpf=no],
+    [#include <linux/bpf.h>]
+)
+
 AC_MSG_CHECKING([if ebpf.plugin should be enabled])
-if test "${build_target}" == "linux" -a "${EBPF_LIBS}"; then
+if test "${build_target}" == "linux" -a \
```

Comment:
I think we should remove all string comparisons with `==` from `configure.ac` since it's not standard, and use `=` instead.

https://github.com/koalaman/shellcheck/wiki/SC2039#testing-equality

Reply:
There are 7 places across `configure.ac` where `==` is used. I think we should do it in a separate PR.